### ISSUE TITLE
Document yarn install --network-timeout

### ISSUE THREAD

**Issue by MartinMuzatko** [yarnpkg/website#1037]
https://yarnpkg.com/lang/en/docs/cli/install/

The option --network-timeout is available to the CLI but not documented.

Thanks in advance.

**Comment by Pixelbog** [comment]
Yeah pretty weird. You can do a `yarn help install` and find the line 
```
--network-timeout <milliseconds>    TCP timeout for network requests
 ```
But I couldn't find it documented anywhere...
